internal/cloudtools/installation.go

```diff
@@ -9,11 +9,8 @@ import (
 // GetInstallationIDFromOwnerID returns the ID of an installation that matches
 // a given OwnerID. Multiple matches will return an error. No match will return
 // an empty ID and no error.
-func GetInstallationIDFromOwnerID(serverURL, awsAPIKey, ownerID string) (string, string, error) {
-	headers := map[string]string{
-		"x-api-key": awsAPIKey,
-	}
-	cloudClient := cloudModel.NewClientWithHeaders(serverURL, headers)
+func GetInstallationIDFromOwnerID(serverURL, ownerID string) (string, string, error) {
+	cloudClient := cloudModel.NewClient(serverURL)
```

Comment:
why remove this? we need to talk via the proxy

Reply:
Would you care to explain that proxy part in more detail? Probably I overstepped my refactoring but AFAIK the client gets the variables from environment so there were no need to use the headers. I asked around this and I couldn't get an answer so I thought it was kind of legacy code

Reply:
Sorry for the delay @ethervoid 
Matterwick is communicating to provisioner via a proxy auth API, which in this case we use AWS API gateway, and to send a request we need to have the header set with the API key., removing that this will not work anymore

Reply:
we can talk more on the channel if you want

Reply:
Understood, than I'll revert those changes. Thank you for the explanation :)

Reply:
Reverted